multi-agent/lib/mas/prompts/service.py

```
@@ -0,0 +1,309 @@
+
+class PromptPermissionError(Exception):
+    def __init__(self, prompt_id: str, identity: Identity):
+        self.prompt_id = prompt_id
+        self.identity = identity
+        super().__init__(
+            f"Identity {identity.id} does not own prompt {prompt_id}"
+        )
+
+
+class PromptService:
+    def __init__(
```

Ruff (0.15.21)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)